BootloaderCommonPkg/Include/Library/ConsoleOutLib.h

```diff
@@ -12,6 +12,70 @@
 #include <PiPei.h>
 #include <Library/BaseLib.h>
 
+//
+// NOT RECOMMEND to use __CONSOLE_DEBUG directly
+//
+#define __CONSOLE_DEBUG_FUNC(Format, ...)                               \
+  do {                                                                  \
+    ConsoleDebugPrint (Format, ##__VA_ARGS__);                          \
+  } while (FALSE)
+
+#define __CONSOLE_DEBUG_UNICODE_FUNC(Format, ...)                       \
+  do {                                                                  \
+    ConsoleDebugPrintUnicode (Format, ##__VA_ARGS__);                   \
+  } while (FALSE)
+
+#define __CONSOLE_DEBUG_CONDITION_FUNC(PrintLevel, Format, ...)         \
+  do {                                                                  \
+    ConsoleDebugPrint (Format, ##__VA_ARGS__);                          \
+  } while (FALSE)
+
+#define __CONSOLE_DEBUG_UNICODE_CONDITION_FUNC(PrintLevel, Format, ...) \
+  do {                                                                  \
+    ConsoleDebugPrintUnicode (Format, ##__VA_ARGS__);                   \
+  } while (FALSE)
+
+#define __CONSOLE_DEBUG_CONDITION(Expression)                           \
+  __CONSOLE_DEBUG_CONDITION_FUNC          Expression
+
+#define __CONSOLE_DEBUG_UNICODE_CONDITION(Expression)                   \
+  __CONSOLE_DEBUG_UNICODE_CONDITION_FUNC  Expression
+//
+// NOT RECOMMEND to use __CONSOLE_DEBUG directly
+//
+
+//
+// CONSOLE_DEBUG will redirect log messages to consoles
+// Expression does not have PrintLevel(ErrorLevel).
+//
+#define CONSOLE_DEBUG(Expression)                                       \

```

Comment:
call CONSOLE_PRINT ?   It is nothing to do with debug.

Reply:
Thanks. It looks CONSOLE_PRINT is better naming. Let me rename it.